pants_plugins/sendwave/pants_docker/target.py

```diff
@@ -57,6 +58,33 @@ class Registry(StringField):
     help = "The registry of the resulting docker image"
 
 
+class CacheFrom(StringField):
+    alias = "cache_from"
+    required = False
+    default = None
+    help = (
+        "Specify a registry that will be used for caching via the "
+        "docker build --cache-from option (requires BuildKit to be installed)"
+    )
+
+
+class BuildkitInlineCache(BoolField):
+    alias = "buildkit_inline_cache"
+    required = False
+    default = False
+    help = (
+        "Enables writing an inline layer cache for the image via the docker build --build-arg "
+        "BUILDKIT_INLINE_CACHE=1 parameter (requires BuildKit to be installed)"
+    )
+
+
+class MultilinePipInstall(BoolField):
```

Comment:
this... might make sense as a subsystem option? but idk?

Reply:
Will do 👍🏻